src/docs/admin/references/NodeSourceInfrastructureReference.adoc

```diff
@@ -283,10 +284,20 @@ me=propertyvalue`). This is an optional property.
 
 -   *nodeTimeout*: The estimated startup time of the nodes (expressed in millisecond). After this timeout expired, the node will be considered as lost.
 
+-   *startupScript*: VM startup script to launch the ProActive nodes. 
+        Usually, this script is expected to download and install the Java Runtime Environment, then download and launch ProActive node.jar, if not already provided by the VM image.
```

Comment:
Suggestion: This script should launch ProActive node.jar. It shis also expected to download and install the Java Runtime Environment, and download and launch ProActive node.jar, if they are not already provided by the VM image.

Reply:
Thanks, I change it to  "To run the ProActive nodes, this script is also expected to download and install the Java Runtime Environment, then download and and execute ProActive node.jar, if they are not already provided by the VM image."